core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -449,7 +449,7 @@ protected function row($strPalette=null)
 			$key = (Input::get('act') == 'editAll') ? 'FORM_FIELDS_' . $suffix : 'FORM_FIELDS';
 
 			// Calculate the current palette
-			$postPaletteFields = implode(',', Input::post($key));
+			$postPaletteFields = implode(',', (array) Input::post($key));
```

Comment:
Why would we want to catch this error? It only occurs if the user input is invalid, so the following seems more appropriate to me:

```php
if (!\is_array($post = Input::post($key)) {
    throw new \RuntimeException('Invalid data submitted!');
}

$postPaletteFields = implode(',', $post);
```

WDYT?

Reply:
Idk, maybe one of the selected records did not contain any field in the palette?

Reply:
Can you debug this?

Reply:
I don't think I can, this just randomly appeared in Sentry, I have no idea what users did. But splitting a value that you're not sure is a string is definitly wrong (Rector for example would add a string casting here as well).

Reply:
Well what did the stack trace reported from Sentry look like?

Reply:
The sentry report (including stack trace) is linked in the initial comment 😉

Reply:
Yes, but the public report does not show the POST data.

Reply:
I can't see any post data in the private view either. But it obviously was not an array, maybe you can explain what yuo're looking for?

Reply:
> maybe you can explain what yuo're looking for?

Just some clues on how to reproduce this.